test-data/conda-lock/v6/sources-lock.yml

```diff
@@ -0,0 +1,37 @@
+version: 6
+environments:
+  default:
+    channels:
+      - url: https://conda.anaconda.org/conda-forge/
+    packages:
+      win-64:
+        - conda: child-package
+packages:
+  - conda: child-package
+    name: child-package
+    version: 0.1.0
+    build: pyhbf21a9e_0
+    subdir: noarch
+    depends:
+      - path
+      - git_tag
+      - git_rev
+      - git_branch
+    # This should cause the lock-file to be invalid because `foobar` should be a source package,
+    # but it is installed in the environment as a binary package.
```

Comment:
As far as I can tell, you use this file to check whether it can be correctly parsed.
What is the purpose of this comment then?